src/server/game/Entities/AreaTrigger/AreaTrigger.cpp

```diff
@@ -263,6 +264,9 @@ bool AreaTrigger::Create(AreaTriggerCreatePropertiesId areaTriggerCreateProperti
         }
     }
 
+    if (IsStaticSpawn())
+        setActive(true);
```

Comment:
I'm not sure about this part specifically, but it resolved an issue I ran into where server area triggers keep players in their target list after logout. Because they were still in the target list, `DoActions` was not called on login, and the rest state update field (loaded from db) became out of sync with the player rest flags.

With this here, the player is removed from the area trigger target list on logout, and properly gets the rest flag as a result of the `DoActions` call on login.

Reply:
Okay, I suspected it wasn't an ideal solution. Do you have any suggestions for resolving the issue I outlined with players logging out and staying in the area trigger `_insideUnits`?

Reply:
setActive, but only while there are units inside

Reply:
I think I have this updated to remain active only while units are inside.